### PR TITLE
Database migrator

### PR DESCRIPTION
**Pull request by cymon1997**
Implement database migrator & seeder template using https://github.com/golang-migrate/migrate

### CODE REVIEW

File: files/config/config.yaml [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+  Port: 8000
+
+DB:
+  DSN: "postgres://go_template:root@127.0.0.1:15432/go_template_test?sslmode=disable"
+  Driver: "postgres"
+  MaxOpenConn: 10
+  MaxIdleConn: 10
+
+Migrator:
+  DSN: "postgres://go_template:root@127.0.0.1:15432/go_template_test?sslmode=disable"
```
Comment (cymonevolution) [inline files/config/config.yaml:6]: hi @cymon1997 can we use the same variable as `DB.DSN` ? wdyt

File: files/config/config.yaml [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+  SchemaTableName: "migration_schema"
+  Path: "file://files/db/migrations"
+
+Seeder:
+  DSN: "postgres://go_template:root@127.0.0.1:15432/go_template_test?sslmode=disable"
```
Comment (cymonevolution) [inline files/config/config.yaml:11]: this one too

**Review by progrevo — CHANGES_REQUESTED** [review]
I think it's a good idea if we also generate the models automatically from the db, wdyt?

**Comment by cymon1997** [comment]
> I think it's a good idea if we also generate the models automatically from the db, wdyt?

Sure, I think we should make a separate issue/PR implementation for that.

File: files/config/config.yaml (hunk 1 above)
Comment (cymon1997) [reply files/config/config.yaml:6]: nevermind, will remove the base DB and re-add with the base db client repository instead. thanks!

File: files/config/config.yaml (hunk 2 above)
Comment (cymon1997) [reply files/config/config.yaml:11]: same as above

File: files/config/config.yaml (hunk 1 above)
Comment (cymonevolution) [reply files/config/config.yaml:6]: okay sure

**Review by progrevo — APPROVED** [review]
LGTM!

**Pull request merged**